Some API cannot work well in spring 6, such as org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor#postProcessPropertyValues is removed in spring6

The error log:
```
HOTSWAP AGENT: 10:27:58.224 ERROR (org.hotswap.agent.plugin.spring.reload.XmlsChangedCommand) - Error invoking method
java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.hotswap.agent.plugin.spring.reload.SpringChangedReloadCommand.executeCommand(SpringChangedReloadCommand.java:33)
	at org.hotswap.agent.command.impl.CommandExecutor.run(CommandExecutor.java:43)
Caused by: java.lang.NoSuchMethodError: 'org.springframework.beans.PropertyValues org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(org.springframework.beans.PropertyValues, java.beans.PropertyDescriptor[], java.lang.Object, java.lang.String)'
	at org.hotswap.agent.plugin.spring.core.AutowiredAnnotationProcessor.processSingletonBeanInjection(AutowiredAnnotationProcessor.java:23)
	at org.hotswap.agent.plugin.spring.reload.SpringBeanReload.processAutowiredAnnotationBeans(SpringBeanReload.java:510)
	at org.hotswap.agent.plugin.spring.reload.SpringBeanReload.doReload(SpringBeanReload.java:299)
	at org.hotswap.agent.plugin.spring.reload.SpringBeanReload.reload(SpringBeanReload.java:259)
	at org.hotswap.agent.plugin.spring.reload.SpringChangedAgent.doReload(SpringChangedAgent.java:201)
	at org.hotswap.agent.plugin.spring.reload.SpringChangedAgent.reloadAll(SpringChangedAgent.java:189)
	at org.hotswap.agent.plugin.spring.reload.SpringChangedAgent.reload(SpringChangedAgent.java:140)
	... 6 more
```

The method `org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor#postProcessPropertyValues` is deprecated in spring5, and it is removed in spring6.


Maybe We should add the label spring6/springboot3.

--- a/plugin/hotswap-agent-spring-plugin/src/main/java/org/hotswap/agent/plugin/spring/core/AutowiredAnnotationProcessor.java
+++ b/plugin/hotswap-agent-spring-plugin/src/main/java/org/hotswap/agent/plugin/spring/core/AutowiredAnnotationProcessor.java
@@ -1,9 +1,11 @@
 package org.hotswap.agent.plugin.spring.core;
 
 import org.hotswap.agent.logging.AgentLogger;
+import org.springframework.beans.PropertyValues;
 import org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor;
 import org.springframework.beans.factory.support.DefaultListableBeanFactory;
 
+import java.lang.reflect.InvocationTargetException;
 import java.util.Map;
 
 public class AutowiredAnnotationProcessor {
@@ -11,20 +13,39 @@ public class AutowiredAnnotationProcessor {
 
     public static void processSingletonBeanInjection(DefaultListableBeanFactory beanFactory) {
         try {
-            Map<String, AutowiredAnnotationBeanPostProcessor> postProcessors = beanFactory.getBeansOfType(AutowiredAnnotationBeanPostProcessor.class);
-            if (postProcessors == null || postProcessors.isEmpty()) {
+            Map<String, AutowiredAnnotationBeanPostProcessor> postProcessors = beanFactory.getBeansOfType(
+                AutowiredAnnotationBeanPostProcessor.class);
+            if (postProcessors.isEmpty()) {
                 LOGGER.debug("AutowiredAnnotationProcessor not exist");
                 return;
             }
             AutowiredAnnotationBeanPostProcessor postProcessor = postProcessors.values().iterator().next();
+            boolean postProcessPropertyValuesNotExists = false;
             for (String beanName : beanFactory.getBeanDefinitionNames()) {
                 Object object = beanFactory.getSingleton(beanName);
                 if (object != null) {
-                    postProcessor.postProcessPropertyValues(null, null, object, beanName);
+                    if (postProcessPropertyValuesNotExists) {
+                        // spring 6.x
+                        postProcessor.postProcessProperties(null, object, beanName);
+                        continue;
+                    }
+                    try {
+                        // from spring 3.2.x to 5.x
+                        postProcessor.postProcessPropertyValues(null, null, object, beanName);
+                    } catch (NoSuchMethodError e) {
+                        // spring 6.x
+                        postProcessor.postProcessProperties(null, object, beanName);
+                        postProcessPropertyValuesNotExists = true;
+                    }
                 }
             }
-        } catch (Exception e) {
-            LOGGER.debug("AutowiredAnnotationProcessor maybe not exist", e);
+        } catch (Throwable e) {
+            if (LOGGER.isDebugEnabled()) {
+                LOGGER.debug("AutowiredAnnotationProcessor maybe not exist", e);
+            } else {
+                LOGGER.warning("AutowiredAnnotationProcessor maybe not exist : " + e.getMessage());
+            }
         }
     }
+
 }

